<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-OOAJ2R4" failures="1" tests="1" name="Default test(failed)(failed)" time="0.125" errors="0" timestamp="2021-05-18T10:13:20 IST">
  <testcase classname="screenshotTestForFail.A19_FailedTestCaseOnlyRunTest" name="sub" time="0.021">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
at screenshotTestForFail.A19_FailedTestCaseOnlyRunTest.sub(A19_FailedTestCaseOnlyRunTest.java:14)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- sub -->
  <testcase classname="screenshotTestForFail.A19_FailedTestCaseOnlyRunTest" name="add" time="0.0">
    <ignored/>
  </testcase> <!-- add -->
  <testcase classname="screenshotTestForFail.A19_FailedTestCaseOnlyRunTest" name="div" time="0.0">
    <ignored/>
  </testcase> <!-- div -->
  <testcase classname="screenshotTestForFail.A19_FailedTestCaseOnlyRunTest" name="mul" time="0.0">
    <ignored/>
  </testcase> <!-- mul -->
</testsuite> <!-- Default test(failed)(failed) -->
